### ISSUE TITLE
File rearranging in Book Mode

### ISSUE THREAD

**Issue by eldelacajita** [fabiocolacio/Marker#196]
This is just an idea for the "Book Mode" exploration you guys are doing (which sounds awesome):

One of the problems of working with several numbered files (either from a template or created by you) is changing the order and then re-numbering them. 

It would be awesome if Marker allowed the user to re-arrange the files in the document list/sidebar, to change their order in the final combined document. The current order could be stored in the configuration makefile, and/or the program could automatically append new order numbers to the files when saving, like `01-Filename.md`, `02-Filename.md` so that they keep the right order even when seen outside Marker. 

This would be an awesome addition over the mentioned templates, where (AFAIK) you have to rename the [source files](https://github.com/tompollard/phd_thesis_markdown/tree/master/source) by yourself if you change their order... which at least I do a lot.

**Comment by fabiocolacio** [comment]
Yes I like this idea. I think the sidebar still needs a lot of polish, and I plan to work on that this summer.
